.gitignore

```diff
@@ -17,3 +17,7 @@ distribution/openhabhome/etc/rrd4j/*.rrd
 *~
 features/*/src/main/history
 itemDump.txt
+
+# IntelliJ files
```

Comment:
Please can you remove this change from this PR. If you want to update this file, then I'd recommend submitting it as a separate request.